bin/build

```diff
@@ -44,6 +45,7 @@ else
       echo "build = false"
       echo "launch = false"
       echo -e "[metadata]\npackage_lock_checksum = \"$local_lock_checksum\""
+      echo -e "stack = \"$local_stack\""
```

Comment:
just a minor styling nit: should we either combine this with line above or split that into two lines?

Reply:
Good call, I'll fix that up